src/redux/publisherConfigSlice.ts

```diff
@@ -0,0 +1,96 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from './rootReducer';
+
+type Countries = 'US' | 'CA' | 'BR' | 'PL';
+
+export type AdyenConfiguration = {
+  checkoutReturnUrl: string;
+  myaccountReturnUrl: string;
+  analytics: {
+    enabled: boolean;
+  };
+  paymentMethodsConfiguration: {
+    card: {
+      name: string;
+      billingAddressRequired: boolean;
+      billingAddressMode: string;
+      brands: string[];
+      brandsConfiguration: {
+        visa: {
+          icon: string;
+        };
+      };
+      showBrandIcon: boolean;
+      showBrandsUnderCardNumber: boolean;
+      positionHolderNameOnTop: boolean;
+      billingAddressAllowedCountries: Countries[];
+      minimumExpiryDate: string;
+      autoFocus: boolean;
+    };
+  };
+  locale: string;
+  openFirstPaymentMethod: boolean;
+};
+
+type PublisherConfig = {
+  offerId: string;
+  paymentMethods: {
```

Comment:
I think (not sure if we are doing it already) that we should have there only supported payment methods, so we would have 
```
methodName: 'card' | 'applepay' | 'googlepay' | 'paypal' | 'manual';
paymentGateway:'adyen' | 'paypal' | 'manual'
```

Reply:
maybe hard to achieve as payment methods as fetched in the old way and the we call `dispatch(updatePaymentMethods(validMethodsFromResponse));` , so also different payment methods could be saved here

Reply:
Okay, I have set methodName and paymentGateway to the above types for now :)